### DISCUSSION TITLE
Could you please add more live channels?

### DISCUSSION THREAD

**Discussion by skonudula** [benmoose39/YouTube_to_m3u#6] [Ideas]
This is a good project.

I have a playlist that has a few movie videos and a few news channels. 
Please review and add the missing videos.

https://www.youtube.com/playlist?list=PLDR8yIc7zGG6QK05d88HLzNeuoi7r7xpq&jct=eGsaCdnzGUj0Sm5S1N8YjEJqcF5Vxw

-Sudheer

**Comment by benmoose39** [comment]
Hey Sudheer, I've added the channels.

**Reply by skonudula** [reply]
Thanks for adding the channels

**Reply by benmoose39** [reply]
Join discord [https://discord.gg/dmgYmAEdee](https://discord.gg/dmgYmAEdee) to be notified of the updates!!

**Reply by codezipper** [reply]
Please add following channels please. 
atv Canlı Yayını
https://www.youtube.com/watch?v=K7TWa2SSK-w  
Star TV - Canlı Yayın HD
https://www.youtube.com/watch?v=vKxjd5-KCEc
Show Türk Canlı Yayın
https://www.youtube.com/watch?v=NTqUvrUmoHc
FOX Canlı Yayın
https://www.youtube.com/watch?v=kv_hci_lb4k

**Reply by marcyjok** [reply]
Could you please add following channels:

EWTN Polska | Polish | https://wp.pl.aleteia.org/wp-content/uploads/sites/9/2018/07/web3-tv-ewtn-poland.jpg |
https://www.youtube.com/watch?v=sTrLPvWOCLg
wPolsce  | Polish | https://media.wplm.pl/pictures/2017/06/18/4246/814/a84dac5f9b7e4eda9e73a5790a0594c4.jpeg |
https://www.youtube.com/watch?v=WYF7OE8lbFQ

Media Narodowe | Polish | https://medianarodowe.com/wp-content/uploads/2018/05/logo_mn_2_mini.png |
https://www.youtube.com/watch?v=YlookK5AD6c

thank you

**Reply by benmoose39** [reply]
> Could you please add following channels:
> 
> EWTN Polska | Polish | https://wp.pl.aleteia.org/wp-content/uploads/sites/9/2018/07/web3-tv-ewtn-poland.jpg | https://www.youtube.com/watch?v=sTrLPvWOCLg wPolsce | Polish | https://media.wplm.pl/pictures/2017/06/18/4246/814/a84dac5f9b7e4eda9e73a5790a0594c4.jpeg | https://www.youtube.com/watch?v=WYF7OE8lbFQ
> 
> Media Narodowe | Polish | https://medianarodowe.com/wp-content/uploads/2018/05/logo_mn_2_mini.png | https://www.youtube.com/watch?v=YlookK5AD6c
> 
> thank you

added

**Comment by marcyjok** [comment]
Could you please EDIT following channels:
reason? - links changed

Could you please add following channels:

EWTN Polska | Polish | https://wp.pl.aleteia.org/wp-content/uploads/sites/9/2018/07/web3-tv-ewtn-poland.jpg |
https://www.youtube.com/watch?v=vwjwKbNM-uQ

wPolsce | Polish | https://media.wplm.pl/pictures/2017/06/18/4246/814/a84dac5f9b7e4eda9e73a5790a0594c4.jpeg |
https://www.youtube.com/live/e23l1KpBUw8


Media Narodowe | Polish | https://medianarodowe.com/wp-content/uploads/2018/05/logo_mn_2_mini.png |
https://www.youtube.com/watch?v=xhnumPF0aCE

thank you

**Comment by Remek33** [comment]
Could you please EDIT following channel:  channel wPolsce channel is here 
https://media.wplm.pl/pictures/2017/06/18/4246/814/a84dac5f9b7e4eda9e73a5790a0594c4.jpeg
https://www.youtube.com/watch?v=dJ1tntPpr_A
